client/ts/graph_chart.ts

```diff
@@ -22,19 +25,32 @@ export function generateGraphChart(jsonUrl){
         // .force('collide', d3.forceCollide().radius(nodeRadius*1.2))
         .force('collide', d3.forceCollide())
         .force('link', d3.forceLink().distance(80)) // distance sets the length of each link
-        .force('link', d3.forceLink().id(function(d){ return d.name; }));  // for source and targe value in links not using number based zero but based on customized string
+        .force('link', d3.forceLink().id(function(d:any){ return d.name; }));  // for source and targe value in links not using number based zero but based on customized string
 
     d3.json(jsonUrl.value, function(error, data){
-        console.log("data",data);
+        // These thicker lines make it easier for someone to click on a graph edge.
+        let bg_links = svg.append("g")
```

Comment:
could you explain the idea of having two links?

Reply:
There are two layers of links. One is wider than the other. The wider line is there to increase the click area of the edge. It is also transparent. The thinner one is the one you see. It does not respond to mouse events.